samples/betaFeatures.js

```diff
@@ -0,0 +1,540 @@
+/**
+ * Copyright 2017, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This application demonstrates how to perform basic recognize operations with
+ * with the Google Cloud Speech API.
+ *
+ * For more information, see the README.md under /speech and the documentation
+ * at https://cloud.google.com/speech/docs.
+ */
+
+'use strict';
+
+function speechTranscribeDiarization(fileName) {
+  //[START speech_transcribe_diarization]
+  const fs = require('fs');
+
+  // Imports the Google Cloud client library
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const fileName = 'Local path to audio file, e.g. /path/to/audio.raw';
+
+  const config = {
+    encoding: `LINEAR16`,
+    sampleRateHertz: 8000,
+    languageCode: `en-US`,
+    enableSpeakerDiarization: true,
+    diarizationSpeakerCount: 2,
+    model: `phone_call`,
+  };
+
+  const audio = {
+    content: fs.readFileSync(fileName).toString('base64'),
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      console.log(`Transcription: ${transcription}`);
+      console.log(`Speaker Diarization:`);
+      const chunks = response.results.map(result => result.alternatives[0]);
+      console.log(`WORDS: `);
+      const lastChunk = chunks[chunks.length - 1].words;
+      lastChunk.forEach(a => console.log(JSON.stringify(a, null, 4)));
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  //[END speech_transcribe_diarization]
+}
+
+function asyncSpeechTranscribeDiarizationGCS(gcsUri) {
+  // [START speech_transcribe_diarization_gcs]
```

Comment:
Is a GCS version requested? I don't see it in the [Python canonicals](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/speech/cloud-client/beta_snippets.py). Double check with canonical author?

Here are all of the region tags in the [Python `beta_snippets.py`](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/speech/cloud-client/beta_snippets.py)

 - `speech_transcribe_diarization`
 - `speech_transcribe_file_with_auto_punctuation`
 - `speech_transcribe_file_with_enhanced_model`
 - `speech_transcribe_file_with_metadata`
 - `speech_transcribe_multichannel`
 - `speech_transcribe_multilanguage`
 - `speech_transcribe_word_level_confidence`

Reply:
Meh, nevermind, I see the GCS versions in the Java samples ([Recognize.java](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/speech/cloud-client/src/main/java/com/example/speech/Recognize.java))